doc/installing.rst

```diff
@@ -52,8 +52,8 @@ create a local environment and install `joblib` in it. One of the pros of
 this method is that you never have to become administrator, and thus all
 the changes are local to your account and easy to clean up.
 
-* **If you are under Pyton 2.6 or above**
-  
+* **If you are under Python 2.6 or above**
```

Comment:
> Done. Appart from that, I think we should add the pip install way.

My bad my first comment was slightly wrong. As I tried to say in my last comment, I would just keep the section and just remove the "If you are under ..." mention since we can assume it is always satisfied.

About adding pip install it may make sense although I wouldn't say it is super high priority. Maybe in a separate PR so we can merge this one quickly.